doc/final_report.md

```diff
@@ -108,32 +108,6 @@ From the different neighbourhoods, Manhattan is the most expensive,
 while the Bronx is the cheapest. We expect that neighbourhood group and
 type of room will be important features in predicting price.
 
-We summarize the differences in a room type in a certain neighbourhood
```

Comment:
I'm not sure if it's a good idea to remove the table. I think it's a part of the requirement from the beginning. Firas suggested a format like [this](https://cran.r-project.org/web/packages/kableExtra/vignettes/awesome_table_in_html.html#add_header_rows_to_group_columns). I think that's valid. I will leave that up to you.

Reply:
I talked to Firas about it and he said its redundant because we show the information in the plots above. I'm going to stick with removing it